WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewModel.swift

```diff
@@ -1,34 +1,78 @@
 
-class ReaderTabViewModel {
-
-    var tabSelectionCallback: ((ReaderAbstractTopic?) -> Void)?
-    var selectedFilter: ReaderAbstractTopic?
+@objc class ReaderTabViewModel: NSObject {
+    // MARK: - Properties
+    /// tab selection
```

Comment:
Could these doc comments have a little more detail?

(and perhaps we could introduce some extra newlines in here to group items / space things out a little)